packages/styleguide/src/components/NavigationBar/index.js

```diff
@@ -3,27 +3,39 @@ import styled from 'styled-components';
 import { bool } from 'prop-types';
 import cx from 'classnames';
 
-import { rem } from './../../style/utils';
+import { sizes } from '../../style/theme';
 
 const propTypes = {
   isActive: bool,
-  isMobileButton: bool,
 };
 
-const NavigationButton = ({
-  className,
-  isActive,
-  isMobileButton = false,
-  ...other
-}) => {
-  const classes = cx({ 'is-active': isActive }, className);
+let previousScrollPosition = window.pageYOffset;
+
+window.onscroll = function() {
+  const currentScrollPosition = window.pageYOffset;
+  const navigationBar = document.querySelector('.navigation-bar');
```

Comment:
Opat toto by som riesil cez referenciu. A pravdepodobne v componentDidMount, ak nie cez hook. Teraz je to taky zvlastny sideffect tohto komponentu, co nie je dobry pattern v React-e.

Reply:
Napr. https://dev.to/n8tb1t/tracking-scroll-position-with-react-hooks-3bbj